src/screens/LoginView/LoginView.jsx

```diff
@@ -6,43 +6,51 @@ import TextBox from '../../components/TextBox/TextBox';
 import CustomButton from '../../components/CustomButton/CustomButton';
 import InputLabel from '../../components/InputLabel/InputLabel';
 import {styles} from '../RegisterView/RegisterViewStyle';
+import { firebaseLogin } from '../../library/methods/firebaseLogin';
+import Spinner from '../../components/Spinner/Spinner';
 
 const LoginView = ({navigation}) => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const [loading, setLoading] = useState(false);
 
-  const {validate, isFieldInError, getErrorsInField, getErrorMessages} = useValidation({state: { email, password }});
+  const {validate, isFieldInError, getErrorsInField} = useValidation({state: { email, password }});
 
-  const onPressButton = () => {
+  const onPressButton = async () => {
     validate({
       email: { email: true, required: true },
       password: { minlength: 8, required: true },
-    });
+    });   
```

Comment:
There are unnecessary spaces here.

```suggestion
    });
```